string     CommandMode
string     Coordinator
string     ZeroAxis
string     Aux_swing
string     Aux_Home
float64[]  CtrlValues
int8 CommandEmit
int8 ManualSwitch
